src/tests/system/tests/test_ldap_extra_attrs.py

```diff
@@ -0,0 +1,57 @@
+from __future__ import annotations
+
+import pytest
+from sssd_test_framework.roles.client import Client
+from sssd_test_framework.roles.generic import GenericProvider
+from sssd_test_framework.topology import KnownTopologyGroup
+
+
+@pytest.mark.tier(1)
+@pytest.mark.ticket(gh=4153, bz=1362023)
+@pytest.mark.topology(KnownTopologyGroup.AnyProvider)
+def test_ldap_extra_attrs__contains_mail(client: Client, provider: GenericProvider):
+    """
+    :title: SSSD fails to start when ldap_user_extra_attrs contains mail
```

Comment:
I want to update the test naming convention, to something that describes the feature, function, and parameter that it's testing when it's a tier1/gating test. I think this falls into this category. We can keep the BZ title in the description. 

I need to run this by the QE team in next week's meeting, which will happen in another PR, other than Pavel's comments, LG2M.

Reply:
We aren't in a hurry here so we can wait until you have this discussion to update the test name

Reply:
We didn't have the discussion last week because several people were on PTO. We will have it during our QE meeting today.

Reply:
I suppose this is not a blocker for this PR. If you want to change the title, then it would have to be done with all tests.

Reply:
Not a blocker, needs to be done on all tests.